### DISCUSSION TITLE
会考虑做多设备同步吗

### DISCUSSION THREAD

**Discussion by leatrise** [shion-app/shion#62] [Ideas]
有多端使用的情况，但是又想两边不分开记录，会不会考虑多端做个同步啊，或者steam版的先考虑一下用云存档？成为独占功能也行

**Comment by hanaTsuk1** [comment]
不考虑
steam的api只能在客户端打开时调用，软件开机自启动不需要打开steam
没服务器做数据合并很麻烦，云存档没那么好用

**Reply by leatrise** [reply]
好的吧，那能不能说一下软件数据位置，我用第三方工具做多端的一个实时同步吧

**Reply by hanaTsuk1** [reply]
`C:\Users\{换成你的用户名}\AppData\Roaming\hanaTsuk1.shion`
希望你已经知晓这可能带来的风险

**Reply by leatrise** [reply]
好的好的
